scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -219,5 +287,26 @@ <h5 class="modal-title">Remove gene panel "{{panel.display_name}}"?</h5>
       $(sel).fadeToggle();
     }
 });
+
+$(function(){
+console.log("called!")
+      function getNameTerms(query, process) {
+        $.get("{{ url_for('genes.api_genes') }}", {query: query}, function(data) {
+          process(data)
+        });
+      }
+
+      $(".typeahead_gene").typeahead({
+        name: 'dynamic_gene_add',
+        source: getNameTerms,
+        minLength: 3,
+      });
+
+      $('select[multiple]').selectpicker({
```

Comment:
I don't think you need this one: you don't have any multiple selects in the form of this page

Reply:
I'm not an expert in javascript. But I interpret `$('select[multiple]').selectpicker'` as being a dynamic index, not allowing for multiple selects in a dropdown. Changing the code to `$('select[i]').selectpicker` has no apparent effect on the page's behaviour.

Reply:
Well, removing it completely changed nothing either. 

Done.